architecture/crypto.rst

```diff
@@ -169,12 +170,37 @@ are the main things to consider when adding a new crypto provider:
     - If you intend to make some algorithms optional, please try to re-use the
       same names for configuration variables as the default implementation.
 
+[5] Support for crypto IC
+*************************
+Some cryptographic co-processors and secure elements are supported under a
+Generic Cryptographic Driver interface, connecting the TEE Crypto generic APIs
+to the HW driver interface. This interface is in
+`core/drivers/crypto/crypto_api`_ and should be followed when adding support for
+new devices.
+
+At the time of writing, OP-TEE does not support the `GP TEE Secure Element API`_
+and therefore the access to the secure element - the NXP EdgeLock® SE05x -
+follows the Cryptographic Operations API presenting a single session to the
+device. This session is shared with the normal world through the PKCS#11
+interface but also through a more generic interface (`libseetec`_) which allows
+clients to send Application Protocol Data Units (APDUs) directly to the device.
+
+Notice that cryptographic co-processors do not necessarily comply with all the
+GP requirements tested and covered by the OP-TEE sanity test suite
+(`optee_test`_). In those cases where the cryptographic operations are not
+supported - i.e: the SE05x does not implement all RSA key sizes - we opted for
+disabling those particular tests at build time rather than letting them fail.
+
```

Comment:
Empty line not needed.

Reply:
um, without that empty line, the hyperlinks break

Reply:
Fair enough, then it's needed.